### PR TITLE
Style footer

### PR DESCRIPTION
**Pull request by shreyshreyansh**
Screenshot 2021-07-28 095621

### CODE REVIEW

File: website/src/components/Footer/Footer.js [hunk 1]
```diff
@@ -1,75 +1,87 @@
...
-        <div className="container">
-          <div className="row">
-            <Scroll showBelow={280} />
+    <footer class="site-footer">
+      <div className="container">
+        <div className="row">
+          <div className="col-sm-12 col-md-6">
+            <h6>About</h6>
+            <p className="text-justify">
+              Women in Technology empowers women and young girls wanting to
```
Comment (shreyshreyansh) [inline website/src/components/Footer/Footer.js:18]: made the changes as said

Comment (shreyshreyansh) [inline website/src/components/Footer/Footer.js:18]: made changes

File: website/src/components/Footer/Footer.css [hunk 2]
```diff
@@ -1,64 +1,134 @@
...
-    }
-}
-
-
-.row1{
-    font-size: 10px;
-}
\ No newline at end of file
+.site-footer {
+  background-color: #ffcc66;
```
Comment (shreyshreyansh) [inline website/src/components/Footer/Footer.css:2]: changed the background color

File: website/src/components/Footer/Footer.css [hunk 3]
```diff
@@ -1,64 +1,134 @@
...
+  padding: 45px 0 20px;
+  font-size: 15px;
+  line-height: 24px;
+  color: #000;
```
Comment (shreyshreyansh) [inline website/src/components/Footer/Footer.css:6]: changed the text color to black

**Review by nehakalbande — APPROVED** [review]
Merging pr #114 @shreyshreyansh 🎉

**Pull request merged**